packages/popover/src/Arrow.js

```diff
@@ -53,9 +52,10 @@ const ArrowAlignment = () =>
     }
   `
 
-const ArrowPlacement = ({ background, borderColor }) => {
-  const bgColor = theme.colors[background]
-  const brColor = theme.colors[borderColor]
+const ArrowPlacement = ({ color, borderColor, ...props }) => {
+  let backgroundColorName = color.indexOf('.') !== -1 ? color : `${color}.base`
```

Comment:
instead you can do this:
```
getPaletteColor(color, 'base')(props)
```
https://priceline.github.io/design-system/utilities/getPaletteColor#palette-color-driven-from-other-prop

Reply:
Oh awesome, thank you!

Reply:
Fixed!